packages/documentation/src/pages/documentation-guide/doc-page.mdx

```diff
@@ -13,67 +13,6 @@ This makes writing long-form content with components a blast 🚀
 
 [https://mdxjs.com/](https://mdxjs.com/)
 
-### Adding React Component documentation
-Adding React Components documentation in Formation.
-
-- **Take a look at the example MDX file at formation-react/src/components/AcceptTermsPrompt/AcceptTermsPrompt.mdx.
-  You can use this as a starter**
-
-- **Create a .mdx file the same name as your component in the `formation-react/src/components` folder, adjacent to your component.**
-    ```
-    - components
-      - YourComponent.js
-      - YourComponent.mdx
-    ```
-- **Add the required frontmatter at the top of the page**
-  ```
-    ---
-    title: AcceptTermsPrompt
-    name: AcceptTermsPrompt
-    ---
-  ```
-- **Restart the development server
-- **If you run into any errors or your page doesn't render correctly,
-  try to delete the .cache file and restart the server again**
-
-- **After you have your page setup you can add markdown and your component code.**
-
-- **Add descriptions to your PropTypes in your Component files**
-
-  We need to convert the propType comments to jsDoc format.
-  This will not show up in the propTypes definitions on your page if you
-  don't have it in a jsDoc format.
-
-  Here is an examples of how it should look:
-
-  Changed code:
-  ```javascript
-    YourComponent.propTypes = {
-      /**
-       * content shown as the actual terms and conditions
-       */
-      user: PropTypes.object,
-      /**
-       * cancel button click handler
-       */
-      onCancel: PropTypes.func
-    };
-  ```
-
-  It will look like this on the page.
-
-  ![proptypes](../../../images/proptypes.png "PropType Image")
```

Comment:
I'm removing this image because this appears to be the only place it was used:

![image](https://user-images.githubusercontent.com/2008881/105897358-e67f4680-5fcc-11eb-99c9-8829420ef761.png)


Additionally, it wasn't even loading on the page properly:

![image](https://user-images.githubusercontent.com/2008881/105897498-19293f00-5fcd-11eb-99bd-a858f9f63709.png)